src/device/router_device_worker.erl

```diff
@@ -1612,7 +1612,7 @@ handle_frame_timeout(
                 time = TxTime,
                 datr = TxDataRate,
                 freq = TxFreq
-            } = lorawan_mac_region:rx1_window(
+            } = lorawan_mac_region:rx1_or_rx2_window(
```

Comment:
Does anything need to change for calculating the `rx2` on the next couple of lines?